app/models/skin.rb

```diff
@@ -188,6 +188,7 @@ def cache!
     self.public = true
     self.official = true
     save!
+    Rails.cache.increment('skins_generation')
     css_to_cache = ""
```

Comment:
help, I can't figure out what is going on with this skins_generation cache. AFAICS it is getting incremented in loads of places and never decremented, so it will endlessly be getting bigger, and it is used only in the cache key? Is the idea that all skin caches should be expired every time any skin is recached or...?